CHANGES.rst

```diff
@@ -156,9 +156,9 @@ See the complete `Gammapy 0.5 merged pull requests list on Github <https://githu
 - [#530] Update readthedocs links from .org to .io (Brigitta Sipocz)
 - [#529] Add data_summary method to DataStore (Johannes King)
 - [#527] Add n-dim data base class for gammapy.irf (Johannes King)
-- [#526] Add King PSF evaluate and to_table_psf methods (Léa Jouvin)
+- [#526] Add King PSF evaluate and to_energy_dependent_table_psf methods (Léa Jouvin)
```

Comment:
I always put the exact PR title as changelog entry.
So `CHANGES.rst` shouldn't be updated going back.
Can you please revert that change?

(Yes, things get outdated, many things will get better names over time. We shouldn't try to rewrite the changelog.)